OutOfSchool/OutOfSchool.WebApi/Models/AddressDto.cs

```diff
@@ -30,7 +19,10 @@ public class AddressDto
 
     public double Longitude { get; set; }
 
-    public long? CATOTTGId { get; set; } = Constants.DefaultCityCodeficatorId;
+    [Required(ErrorMessage = "CATOTTGId is required")]
+    public long CATOTTGId { get; set; }
+
+    public CATOTTG CATOTTG { get; set; }
```

Comment:
`AllAddressPartsDto CodeficatorAddressDto` should cover most of the codeficator already.

Reply:
Deleted